conf/services.conf

```diff
@@ -81,8 +81,7 @@ simple_api_endpoints = {
             PhysicalDescription = ^ca_objects.PhysicalDescription,
             Subjects = "["<unit relativeTo="ca_occurrences" delimiter="," restrictToTypes="subject">&quot;^preferred_labels&quot;</unit>"]",
             Summary = ^ca_objects.Summary,
-            Notes = ^ca_objects.Notes,
-            TermsOfUse = ^ca_objects.TermsOfUse
```

Comment:
First thing on page 38, under section 2.5.3, there is no asterisk against "Terms of Use".